registration/api/v2/serializers.py

```diff
@@ -315,6 +316,37 @@ def validate_user(self, value):
         else:
             return value
 
+    # def validate_external_domain(self, value):
+    #     """
+    #     Prevent user from using a bad domain.
+    #
+    #     Bad domains are understood as:
+    #         - OpenCraft domain
+    #         - Domain in use or such that it is a subdomain of an existing domain
+    #         - Domain such that, when studio, ecommerce and preview subdomains are created, they
+    #         override a domain in use
+    #     """
+    #     is_opencraft_domain = BetaTestApplication.BASE_DOMAIN == value
+    #     if is_opencraft_domain:
+    #         raise ValidationError("Can not use OpenCraft domain.")
+    #
+    #     _, _, domain = value.partition('.')
+    #     is_domain_in_use = BetaTestApplication.objects.filter(external_domain=domain).exists()
```

Comment:
We need to check all OpenEdxInstances, not just the betatest applications - ie. all these fields:

 https://github.com/open-craft/opencraft/blob/cd0595d19effdb1b14cd5aaec6c3f22b6c11a50a/instance/models/mixins/domain_names.py#L83-L88